modules/ROOT/pages/email/email-list.adoc

```diff
@@ -76,9 +76,9 @@ Here is the basic syntax for listing the emails in a mailbox:
 <email:list-imap config-ref="imap-config" mailboxFolder="INBOX"/>
 ----
 
-The examples above return all the emails in the configured mailbox servers that reside in the INBOX folder.
+The previous examples return all the emails in the configured mailbox servers that reside in the INBOX folder.
```

Comment:
Does this say that the email messages are in the servers, and the servers are in the INBOX folder?
Is it supposed to say that the email messages are in the INBOX folders of the servers?
I'm just a little confused about what is in what else: mail > folder > server makes the most logical sense to me, but that doesn't mean it's correct in this context.